File: web/library/src/app/routes/articles/article.ts
export class Article {
    id?:number;
    book:string;
    author:string;
    serial?:number;
    title:string;
    article?:string;

    constructor(book:string,author:string,serial:number,title:string,article:string){
        this.book=book;
        this.author=author;
        this.serial=serial;
        this.title=title;
        this.article=article;
    }
}

File: internal/article/pkg/article/article.go
// Package article 文章模块.
package article

// Article 文章模型.
type Article struct {
	ID      int64  `json:"id,omitempty"`
	Author  string `json:"author"`
	Book    string `json:"book"`
	Title   string `json:"title"`
	Content string `json:"content"`
	// Serial章节排序序号.
	Serial float64 `json:"serial"`
}
